website/widgets/content/markup.md

```diff
@@ -24,3 +24,19 @@ menu = '''
 
 q.page['example'] = ui.markup_card(box='1 1 2 2', title='Menu', content=menu)
 ```
+
+## Without padding
+
+You can set the `compact` attribute to `True` to remove the title and padding of a markup card.
+
+```py
+menu = '''
+<ol>
+    <li>Spam</li>
+    <li>Ham</li>
+    <li>Eggs</li>
+</ol>
```

Comment:
Maybe we could use a better example that would showcase feature. `<ol>` has some spacing by default. E.g. `span` should be better.

<img width="330" alt="image" src="https://user-images.githubusercontent.com/64769322/232448154-27e0b108-8da7-493c-a3f7-78cbd13e7d19.png">

Since we only take a screenshot of the card content, not going to be visible in light docs theme, but that's fine for now.